src/schema/objects/metadata.yaml

```diff
@@ -2825,6 +3023,55 @@ ScatterFraction:
     type: number
     minimum: 0
     maximum: 100
+ScreenDistance:
+  name: ScreenDistance
+  display_name: Screen Distance
+  description: |
+    Screen distance in cm
+    (for example `60` for a screen distance of 60 cm), if no screen use `n/a`.
+  anyOf:
+    - type: number
+      unit: cm
+    - type: string
+      enum:
+        - n/a
+ScreenRefreshRate:
+  name: ScreenRefreshRate
+  display_name: Screen Refresh Rate
+  description: |
+    Refresh rate of the screen (when used), in Hertz
+    (for example, `60` for 60 Hz).
+  type: number
+  unit: Hz
+ScreenResolution:
+  name: ScreenResolution
+  display_name: Screen Resolution
+  description: |
+    Screen resolution in pixel
+    (for example `[1920, 1200]` for a screen of 1920-length by 1080-height pixels),
+    if no screen use `n/a`.
+  anyOf:
+    - type: array
+      items:
+        type: number
```

Comment:
Should this be limited to only two values? Also, since the unit is pixels, should it be integers only?

Reply:
Yes and yes